src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-size;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    height: $column-size;
+    width: $column-size;
```

Comment:
Ensure that the variable `$column-size` is defined in your variables file and follows the Code Style Rules for naming variables (should be kebab-case).